src/traverse.js

```diff
@@ -9,7 +9,7 @@ var sequence = require('./sequence');
  * then uses [`sequence`](#sequence) to transform the resulting Traversable of Applicative
  * into an Applicative of Traversable.
  *
- * Dispatches to the `sequence` method of the third argument, if present.
+ * Dispatches to the `traverse` method of the third argument, if present.
```

Comment:
@davidchambers I think no, as [map](https://github.com/ramda/ramda/blob/v0.24.1/src/map.js#L45) is dispatching both, `map` and `fantasy-land/map` but docs are mentioning just `map` method.
As I correctly understand, we need to dispatch `traverse` method along with `fantasy-land/traverse` like it's done in `map`